contrib/systemd-with-workers/system/matrix-synapse-worker@.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Synapse Matrix Worker
+After=matrix-synapse.service
+BindsTo=matrix-synapse.service
```

Comment:
Does the *BindsTo* enforce the *matrix-synapse.service* gets started, too? This seems to be bad when a worker is running on a host different from the Synapse service. Or do I have to start the Synapse process on each host running any worker?